hyperspy/tests/drawing/test_figure_title.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2007-2016 The HyperSpy developers
```

Comment:
This file should be removed, most likely a leftover from the resolution of a merge conflict.

Reply:
Yes a leftover file, now removed